dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -18,6 +18,15 @@ import 'package:path/path.dart' as path;
 /// adding Flutter to an existing iOS app.
 Future<void> main() async {
   await task(() async {
+    // Update pod repo.
```

Comment:
```
[module_test_ios] [STDOUT] stdout:                [!] CocoaPods could not find compatible versions for pod "GoogleSignIn":
[module_test_ios] [STDOUT] stdout:                  In Podfile:
[module_test_ios] [STDOUT] stdout:                    google_sign_in_ios (from `.symlinks/plugins/google_sign_in_ios/ios`) was resolved to 0.0.1, which depends on
[module_test_ios] [STDOUT] stdout:                      GoogleSignIn (~> 6.2)
[module_test_ios] [STDOUT] stdout: 
[module_test_ios] [STDOUT] stdout:                None of your spec sources contain a spec satisfying the dependency: `GoogleSignIn (~> 6.2)`.
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8802907618391831937/+/u/run_module_test_ios/test_stdout

Reply:
Thanks. I should start inline commenting failing builds on my fix PRs.